plugins/transforms/workflowexecutor/src/main/samples/workflowexecutor/pl_Workflow Executor Sample.hpl

```diff
@@ -0,0 +1,195 @@
+    <created_user>-</created_user>
+    <created_date>2021/04/27 12:31:17.218</created_date>
+    <modified_user>-</modified_user>
+    <modified_date>2021/04/27 12:31:17.218</modified_date>
+    <key_for_session_key>H4sIAAAAAAAAAAMAAAAAAAAAAAA=</key_for_session_key>
+    <is_key_private>N</is_key_private>
+  </info>
+  <notepads>
+  </notepads>
+  <order>
+    <hop>
+      <from>pass parameters</from>
+      <to>wf_get_list_of_transformations</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>wf_get_list_of_transformations</from>
+      <to>Result files</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>wf_get_list_of_transformations</from>
+      <to>Result rows</to>
+      <enabled>Y</enabled>
+    </hop>
+    <hop>
+      <from>wf_get_list_of_transformations</from>
+      <to>Results</to>
+      <enabled>Y</enabled>
+    </hop>
+  </order>
+  <transform>
+    <name>Result files</name>
+    <type>Dummy</type>
+    <description/>
+    <distribute>Y</distribute>
+    <custom_distribution/>
+    <copies>1</copies>
+    <partitioning>
+      <method>none</method>
+      <schema_name/>
+    </partitioning>
+    <attributes/>
+    <GUI>
+      <xloc>864</xloc>
+      <yloc>176</yloc>
+    </GUI>
+  </transform>
+  <transform>
+    <name>Result rows</name>
+    <type>Dummy</type>
+    <description/>
+    <distribute>Y</distribute>
+    <custom_distribution/>
+    <copies>1</copies>
+    <partitioning>
+      <method>none</method>
+      <schema_name/>
+    </partitioning>
+    <attributes/>
+    <GUI>
+      <xloc>864</xloc>
+      <yloc>272</yloc>
+    </GUI>
+  </transform>
+  <transform>
+    <name>Results</name>
+    <type>Dummy</type>
+    <description/>
+    <distribute>Y</distribute>
+    <custom_distribution/>
+    <copies>1</copies>
+    <partitioning>
+      <method>none</method>
+      <schema_name/>
+    </partitioning>
+    <attributes/>
+    <GUI>
+      <xloc>864</xloc>
+      <yloc>64</yloc>
+    </GUI>
+  </transform>
+  <transform>
+    <name>pass parameters</name>
+    <type>DataGrid</type>
+    <description/>
+    <distribute>Y</distribute>
+    <custom_distribution/>
+    <copies>1</copies>
+    <partitioning>
+      <method>none</method>
+      <schema_name/>
+    </partitioning>
+    <fields>
+      <field>
+        <name>parameter</name>
+        <type>String</type>
+        <format/>
+        <currency/>
+        <decimal/>
+        <group/>
+        <length>-1</length>
+        <precision>-1</precision>
+        <set_empty_string>N</set_empty_string>
+      </field>
+    </fields>
+    <data>
+      <line>
+        <item>one</item>
+      </line>
+      <line>
+        <item>two</item>
+      </line>
+      <line>
+        <item>three</item>
+      </line>
+    </data>
+    <attributes/>
+    <GUI>
+      <xloc>361</xloc>
+      <yloc>177</yloc>
+    </GUI>
+  </transform>
+  <transform>
+    <name>wf_get_list_of_transformations</name>
```

Comment:
should be "wf_get_list_of_pipelines"

Reply:
corrected <wf_get_list_of_transformations> with <wf_get_list_of_pipeliens>